libr/util/signal.c

```diff
@@ -16,7 +16,10 @@ static struct {
 	{ "SIGSEGV", SIGSEGV },
 	{ "SIGTERM", SIGTERM },
 #if __linux__
+#if __mips__
+#else
 	{ "SIGSTKFLT", SIGSTKFLT },
```

Comment:
Just remove this line instead of ifdefing.

i dont thibk this signal name is used anywhere

Reply:
okay